test/e2e/scheduling/preemption.go

```diff
@@ -426,18 +431,12 @@ var _ = SIGDescribe("SchedulerPreemption", framework.WithSerial(), func() {
 			}))
 		}
 
-		ginkgo.By("Remove the finalizer from all low priority pods to proceed the preemption.")
-		for _, pod := range lowPriorityPods {
-			// Remove the finalizer so that the pod can be deleted by GC
-			e2epod.NewPodClient(f).RemoveFinalizer(ctx, pod.Name, testFinalizer)
-		}
-
 		ginkgo.By("Wait for high priority pods to be scheduled.")
```

Comment:
Would it be faster if we forcibly remove pods before this line? (where we were removing the finalizer)
Otherwise, here we have to spend ~80s to wait for low priority pods to be removed, right?

Reply:
Yeah, you are right. There's a lot of low priority pods waiting for the TerminationGracePeriodSeconds to pass.